doc/inlining.md

```diff
@@ -51,75 +80,128 @@ to:
 ```
 function main()
   var x = 1
-  var call = someExpression
-  osr [call != 'foo] ...
-  call y = 'foo(x)
+  var fun = someExpression
+  assume [fun == 'foo] else (main, base, lcall) [var x = x, var fun = fun]
+  lcall: call y = 'foo(x) -> l
   print y
 ```
 
 ## With osr
 
-Having the osr in the outer function poses no additional difficulties. Now lets assume the inlinee has safepoints.
-As an example:
+Having the osr in the outer function poses no additional
+difficulties. Now lets assume the inlinee has safepoints.
+
+As an example (we use `rN` for "calle**r** labels", `eN` for
+"calle**e** label", and `iN` for "**i**nlined-code" labels):
```

Comment:
Markdown fails to render this correctly in diff mode (I hoped that the quote I added would fix it), but it renders correctly otherwise as a markdown document, so let's go for that.